### PR TITLE
remove all unnecessary Dockerfiles from repo

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Comment by cesnietor** [comment]
We need to remove also:
`.dockerignore`
Also update https://github.com/minio/console/blob/master/DEVELOPMENT.md since it has some stale docker docs.

**Comment by cesnietor** [comment]
@harshavardhana  did you do `make assets`? there's no need to do that, we do that only during releases.

**Comment by harshavardhana** [comment]
> @harshavardhana did you do `make assets`? there's no need to do that, we do that only during releases.

was trying to the pdfjs-dist fixed but looks like that's not the right way to do it.

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -1,191 +0,0 @@
...
-...
-```
-
-### 3. Build `MinIO`
-
-Still in the MinIO folder, run
-
-```shell
-make build
-```
```
Comment (cesnietor) [inline DEVELOPMENT.md:78]: This is still valid and useful for development @harshavardhana

**Review by cesnietor — APPROVED** [review]
LGTM

**Review by bexsoft — APPROVED** [review]
LGTM, UI Vulnerability will be handled in a separate PR

**Pull request merged**